### PR TITLE
WIP: adding OCI output mode as buildctl does

### LINKED ISSUE #604: [REQ]  Add support for other output modes as buildctl does
### What kind of request is this?

None

### What is your request or suggestion?

https://github.com/project-copacetic/copacetic/blob/d648155f5424a9f4cb13acd7209195846791873b/pkg/patch/patch.go#L209

Turning copacetic TODO comments into issues from https://docs.google.com/spreadsheets/d/1XwNj1J6e2FrUhlqaIsV10l8_tgov7WodlkvpNZXYZMU/edit#gid=1386834576.

### Are you willing to submit PRs to contribute to this feature request?

- [ ] Yes, I am willing to implement it.

### PR DESCRIPTION
**Pull request by h4l0gen**
Describe the changes in this pull request using active verbs such as _Add_, _Remove_, _Replace_ ...
This PR has one little issue it is WIP. I am resolving it by discussion with the maintainers
Closes #604

### CODE REVIEW

File: pkg/patch/patch.go [hunk 1]
```diff
@@ -144,20 +151,56 @@ func patchWithContext(ctx context.Context, ch chan error, image, reportFile, pat
...
-		Session:  attachable, // used for authprovider, sshagentprovider and secretprovider
+		Exports:  []client.ExportEntry{},
+		Frontend: "",
+		Session:  attachable,
 	}
+
+	if outputFormat == "docker" {
+        solveOpt.Exports = append(solveOpt.Exports, client.ExportEntry{
+            Type: client.ExporterDocker,
+            Attrs: map[string]string{
```
Comment (ashnamehrotra) [inline pkg/patch/patch.go:162]: For oci, would it be possible to do this similarly using `client.ExporterOCI`

Comment (ashnamehrotra) [reply pkg/patch/patch.go:162]: Once it is working for OCI output it would also be good to add an integration test for this

**Comment by ashnamehrotra** [comment]
@h4l0gen are you still working on this issue?

**Comment by h4l0gen** [comment]
Hi @ashnamehrotra, sorry I am deeply involved in some other project's engagements. Please feel free to assign it to someone who wants to work on it. Again sorry that I am not able to find time ;)

**Comment by ashnamehrotra** [comment]
@h4l0gen no problem, thank you for getting back to me. I will close this PR for now.

**Pull request closed without merging**